nsxt/resource_nsxt_manager_cluster.go

```diff
@@ -356,7 +356,7 @@ func joinNodeToCluster(clusterID string, certSha256Thumbprint string, guestNode
 	}
 	_, err = client.Joincluster(joinClusterParams)
 	if err != nil {
-		return fmt.Errorf("Failed to join node to cluster: %s, node ip address: %s", err, guestNode.IPAddress)
+		return logAPIError(fmt.Sprintf("Failed to join node to cluster: %s, node ip address: %s", clusterID, guestNode.IPAddress), err)
```

Comment:
`Failed to join node` test is repeated on Create (but not on Update), maybe lets remove the duplicate text from Create flow?

Reply:
Yeah makes sense